lib/internal/modules/esm/module_job.js

```diff
@@ -148,17 +176,17 @@ class ModuleJob extends ModuleJobBase {
     const modules = await SafePromiseAllReturnArrayLike(modulePromises);
     this.module.link(specifiers, modules);
 
-    return dependencyJobs;
+    return evaluationDepJobs;
   }
 
-  instantiate() {
+  #instantiate() {
     if (this.instantiated === undefined) {
-      this.instantiated = this._instantiate();
+      this.instantiated = this.#_instantiate();
     }
     return this.instantiated;
   }
 
-  async _instantiate() {
+  async #_instantiate() {
```

Comment:
Nit: # should be a replacement for _.

```suggestion
  async #instantiate() {
```

Reply:
There's already a `#instantiate` defined above